docs/crio.conf.5.md

```diff
@@ -105,6 +105,10 @@ Example:
 **no_pivot**=*true*|*false*
   Instructs the runtime to not use pivot_root, but instead use MS_MOVE
 
+**default_mounts**=[]
+  Default mounts list for the mount paths, in the form host:container, to be mounted
```

Comment:
List of mount points, in the form host:container, to be mounted in every container

Reply:
fixed